app/models/Mortgage.test.ts

```diff
@@ -18,7 +18,7 @@ it("correctly calculates the amount of the mortgage ", () => {
     initialDeposit: 0.1,
   });
 
-  expect(mortgage.principal).toBeCloseTo(90);
+  expect(mortgage.principle).toBeCloseTo(90);
```

Comment:
I think we want this to be `principal` also right?

Reply:
Yes I think so! 

Thanks for the catches, maybe my find-and-replace-all didn't quite work 🤔